### PR TITLE
Upgrading To Laravel 5.0

### PR DESCRIPTION
**Pull request by dawid-z**
Added a new version ``v2.0`` for support **Laravel 5.0.***

##### Usage
Update Your ``composer.json``

```
...
"repositories": [{
	"type": "vcs",
	"url": "https://github.com/dawid-z/laravel-menu"
}],
"require": {
	"laravel/framework": "5.0.*",
	"lavary/laravel-menu": "dev-master"
}
...
```

Call ``composer update``

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1351,7 +1356,7 @@ In this example we name View-1 `custom-menu.blade.php` and View-2 `custom-menu-i
 ```php
 <nav class="navbar">
   <ul class="horizontal-navbar">
-    @include('custom-menu-items', array('items', $MyNavBar->roots()))
+    @include('custom-menu-items', array(items' => $MyNavBar->roots()))
```
Comment (MattWohler) [inline README.md]: looks like you accidentally removed the apostrophe ( ' )

**Comment by dawid-z** [comment]
Yep, I fixed it.

File: README.md (hunk 1 above)
Comment (dawid-z) [inline README.md]: Yep, I fixed it.

**Comment by dustingraham** [comment]
Should this be closed?

**Pull request closed without merging**